app/javascript/components/molecules/LeaseUpSidebar.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import LeaseUpSidebarButtons from './LeaseUpSidebarButtons'
+import StatusItems from './StatusItems'
+
+import StatusItemShape from '../../utils/shapes/StatusItemShape'
+import { LEASE_UP_STATUS_VALUES } from '../../utils/statusUtils'
+
+const LeaseUpSidebar = ({ currentStatus, statusItems, onSaveClicked, onChangeStatus, onAddCommentClicked }) => {
+  const sidebarButtons = (withMobileStyling) => (
+    <div className={withMobileStyling ? 'hide-large-up' : 'show-large-up'}>
+      <LeaseUpSidebarButtons
+        status={currentStatus}
+        withMobileStyling={withMobileStyling}
+        onSaveClicked={onSaveClicked}
+        onChangeStatus={onChangeStatus}
+        onAddCommentClicked={onAddCommentClicked}
+      />
+    </div>
+  )
+
+  return (
+    <div className='sidebar-content'>
+      {sidebarButtons(false)}
+      <h3 className='status-history-label'>Status History</h3>
+      <StatusItems statusItems={statusItems} />
+      {sidebarButtons(true)}
+    </div>
+  )
+}
+
+LeaseUpSidebar.propTypes = {
```

Comment:
I didn't realize that we were using propTypes.

Reply:
@akegan some components use them in partners, some don't. I prefer them but can remove if you don't want to use them. I like it for objects that are passed around, and that you can easily see that the "status" string relates to the list in statusUtils

Reply:
yeah they seem helpful! just hadn't seen us using them before. I'd be up for using them more. Would help with eventual conversion to typescript too

Reply:
Yep I was thinking the same thing, but didn't want to be too idealistic haha